core/quill.js

```diff
@@ -315,12 +315,12 @@ class Quill {
     }, source);
   }
 
-  setSelection(index, length, source) {
+  setSelection(index, length, reversed = false, source) {
```

Comment:
I'd rather not change the API surface at the moment. The way this can be supported is only if users do `setSelection(new Range(0, 1, true));` Alternatively `setSelection(1, -1)` can be accepted and would generate `new Range(0, 1, true)`